__tests__/app.test.js

```diff
@@ -101,12 +113,32 @@ describe("app", () => {
           expect(body.msg).toBe("Article not found");
         });
     });
-    test("status: 400, responds with wrong format", () => {
+    test("status: 400, responds with invalid input when wrong key input", () => {
+      const updateVote = { inc_votest: 1 };
+      return request(app)
+        .patch("/api/articles/3")
+        .send(updateVote)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("invalid input");
+        });
+    });
+    test("status: 400, responds with invalid input when multiple keys input", () => {
```

Comment:
just make sure the increase_votes key isn't misspelled in this test to ensure that it is failing because of the additional keys and not just due to there being no relevant key present.